### ISSUE TITLE
Share button doesn't work

### ISSUE THREAD

**Issue by bartmika** [Oinweb/py-fly#62]
When you succeed at a goal and you go to the complete page, the share button doesn't work.

**Comment by bartmika** [comment]
Also for Quiz Completion page.

**Comment by viralrain** [comment]
Currently, we're using a plugin called share-button that allows the user to share through twitter, facebook, google+ and email.  The only problem is there doesn't seem to be a way to customize the message that gets shared or customize the url that gets shared.  I'm looking around for something to replace it that will look good still and allow to share across all 3 networks.  
Apparently the google +1 button and twitter share buttons that are done by those companies are pretty configurable, so that's where I'm going to look first
For now, the share button is working, but shares the current page the user is on.

**Comment by viralrain** [comment]
I switched the sharing around, so on the badge and level up notifications, the user will see a button that says share which will take them to the publicly shared page.  Then on the publicly shared page, there is a share button that allows sharing through facebook, twitter and google+
I figured this would work a bit better since the share button only shares the page the user is currently on and I haven't been able to find any way to customize the button based on url or message text or anything like that.  We'll just have to add in a popup or something on the public pages that shows up if the user is logged in telling them to click the share button to share with their social media networks.
